packages/editor/src/client/atoms/formControlUrl.tsx

```diff
@@ -12,15 +15,18 @@ interface UrlValidationProps {
 
 export function FormControlUrl({placeholder, name, value, onChange}: UrlValidationProps) {
   const {t} = useTranslation()
-  const [invalidInput, setInvalidInput] = useState(false)
+  const [
```

Comment:
If this is now handled by rsuite, do we want to get rid of this code altogether?